### PR TITLE
Header for Mobile

### PR DESCRIPTION
**Pull request by Chimoha31**
Finished coding my header for mobile

### CODE REVIEW

File: Santouka/css/main.css [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+  /* border: 2px solid red;  */
+  background-color:rgb(46, 45, 45);
+  display: flex;
+}
+
+header img {
+  /* border: 2px solid blue;  */
+  height: 4em;
+}
+
```
Comment (SOunit) [inline Santouka/css/main.css:34]: maybe you should start to use class using approach.
html tags for styling is weak for html changes.

ex.
change tags from h1 to div, then all style will gone....

Comment (SOunit) [reply Santouka/css/main.css:34]: style and tags should be independent

File: Santouka/css/main.css [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+  }
+  
+  body {
+    background-color:rgba(255,255,255, 0.0);
+    background-blend-mode: normal;
+  }
+}
+
+
+
```
Comment (SOunit) [inline Santouka/css/main.css]: you should leave comments about comment out

// FIXME
// all style will be applied in next project

File: Santouka/css/main.scss [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+  transform: 
+      perspective(800px) 
+      scale(1) 
+      translateX(0)
+      rotateY(0);
+  transform-style: preserve-3d;
+
+}
+
+
```
Comment (SOunit) [inline Santouka/css/main.scss:38]: same.
generally, class based approach is better than tag approach.

File: Santouka/index.html [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Santouka</title>
+
+  <link rel="stylesheet" href="css/main.css">
+</head>
+<body>
+  <header>
```
Comment (SOunit) [inline Santouka/index.html]: my approach is

// tag = class
header = main-header
ul = nav-items
li = nav-item
a = nav-item__link
a active = nav-item__link--active

Comment (SOunit) [reply Santouka/index.html]: i always follow BEM style for css naming

File: Santouka/js/main.js [hunk 5]
```diff
@@ -0,0 +1,24 @@
+"use strict";
+{
+  // All, [elem, elem, elem, elem]
```
Comment (SOunit) [inline Santouka/js/main.js]: I need some comment 
1. why they are comment out
2. why they stay here ( fixed in the future? )

**Review by SOunit — CHANGES_REQUESTED** [review]
i added some fix comment here!

File: Santouka/css/main.css (hunk 1 above)
Comment (Chimoha31) [reply Santouka/css/main.css:34]: I fixed!

File: Santouka/css/main.css (hunk 2 above)
Comment (Chimoha31) [reply Santouka/css/main.css]: I deleted it!

File: Santouka/css/main.scss (hunk 3 above)
Comment (Chimoha31) [reply Santouka/css/main.scss:38]: I fixed it !

File: Santouka/index.html (hunk 4 above)
Comment (Chimoha31) [reply Santouka/index.html]: I fixed it!

File: Santouka/js/main.js (hunk 5 above)
Comment (Chimoha31) [reply Santouka/js/main.js]: I deleted it!

**Comment by SOunit** [comment]
You fixed all? great!

if you push all changes to your remote develop, this pull request may find changes.
Or, maybe you have to make pull-request again on your remote develop branch, then this pull-request updated.

Let's see anyway!


#### git instruction is like this.
- Add more commits by pushing to the develop branch on Chimoha31/ciccc_final_ramens.

File: Santouka/css/main.css (hunk 1 above)
Comment (SOunit) [reply Santouka/css/main.css:34]: lgtm

File: Santouka/index.html [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Santouka</title>
+
+  <link rel="stylesheet" href="css/main.css">
+</head>
+<body>
+  <header class="header_container">
+    <div class="nav_items">
```
Comment (SOunit) [inline Santouka/index.html:12]: test

**Review by SOunit — COMMENTED** [review]
lgtm

**Pull request merged**